### PR TITLE
Fix kfctl_second_apply test

### PR DESCRIPTION
**Pull request by richardsliu**
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/kfctl/218)

### CODE REVIEW

File: py/kubeflow/kfctl/testing/pytests/kfctl_second_apply.py [hunk 1]
```diff
@@ -6,7 +6,6 @@
 from kubeflow.kfctl.testing.util import kfctl_go_test_utils as kfctl_util
 from kubeflow.testing import util
 
-
 @pytest.mark.skipif(os.getenv("JOB_TYPE") == "presubmit",
                     reason="test second apply doesn't run in presubmits")
```
Comment (zhenghuiwang) [inline py/kubeflow/kfctl/testing/pytests/kfctl_second_apply.py]: can we remove this now?

**Comment by richardsliu** [comment]
Here's the presubmit where the second_apply test passed: http://testing-argo.kubeflow.org/workflows/kubeflow-test-infra/kubeflow-kfctl-presubmit-kfctl-go-iap-218-cbf79c8-5873-af90?tab=workflow

Maybe we should just remove the skip flag since it is now passing.

File: py/kubeflow/kfctl/testing/pytests/kfctl_second_apply.py (hunk 1 above)
Comment (richardsliu) [reply py/kubeflow/kfctl/testing/pytests/kfctl_second_apply.py]: removed

**Pull request merged**